patches/all/@oclif+plugin-help+3.2.0.patch

```diff
@@ -1,37 +1,37 @@
 diff --git a/node_modules/@oclif/plugin-help/lib/command.js b/node_modules/@oclif/plugin-help/lib/command.js
-index ab72c1b..75078d5 100644
+index 1caa65b..b84a5b5 100644
 --- a/node_modules/@oclif/plugin-help/lib/command.js
 +++ b/node_modules/@oclif/plugin-help/lib/command.js
-@@ -86,7 +86,7 @@ class CommandHelp {
-         if (!args.filter(a => a.description).length)
+@@ -87,7 +87,7 @@ class CommandHelp {
+         if (args.filter(a => a.description).length === 0)
              return;
-         let body = list_1.renderList(args.map(a => {
+         const body = list_1.renderList(args.map(a => {
 -            const name = a.name.toUpperCase();
 +            const name = a.required ? `<${a.name}>` : `[${a.name}]`;
              let description = a.description || '';
              if (a.default)
                  description = `[default: ${a.default}] ${description}`;
-@@ -130,8 +130,7 @@ class CommandHelp {
+@@ -131,8 +131,7 @@ class CommandHelp {
                      value = flag.options.join('|');
                  }
                  if (!value.includes('|'))
 -                    value = underline(value);
 -                left += `=${value}`;
-+                left += ` <${value}>`;
++                    left += ` <${value}>`;
```

Comment:
@srlowe I fixed the indentation here to signify it's part of the `if` statement. I think that's a bug but as I don't know what it's doing I've left the behavior as is and I'll let you take a look